pom.xml

```
@@ -46,6 +52,8 @@
     <roaringbitmap.version>0.6.59</roaringbitmap.version>
     <rxjava2.version>2.1.9</rxjava2.version>
     <slf4j.version>1.7.25</slf4j.version>
+    <sonar.moduleKey>${project.artifactId}</sonar.moduleKey>
+    <sonar.projectKey>org.heigit.ohsome:oshdb</sonar.projectKey>
```

Does something like this work here?
```suggestion
    <sonar.projectKey>${project.groupId}:oshdb</sonar.projectKey>
```

---

let's try